src/main/java/com/google/vinet/data/IsolateTimeSlot.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.vinet.data;
+
+import com.google.appengine.api.datastore.*;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Query;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.*;
+import java.util.stream.*;
+
+public class IsolateTimeSlot extends TimeSlot implements Datastoreable{
```

Comment:
This could use some more documentation.

What is an IsolateTimeSlot? What is it used for? Why do we need it?

Reply:
I've imrpoved the documentation of the Class's purpose now 👍 It's not too much, but it should better contextualise things https://github.com/googleinterns/step226-2020/pull/8/commits/683542cba30bcdf78724d3e27c57922eda1367b1